hspec.cabal

```diff
@@ -32,7 +32,7 @@ description:      Hspec is a testing framework for Haskell. It is inspired by
                   The Hspec Manual is at <http://hspec.github.io/>.
 
 extra-source-files:
-    changelog
+    CHANGES.markdown
```

Comment:
@dcoutts I'm not on freenode anymore. But at the point there is code to look at, I'm happy to give feedback.